examples/cfd/01_convection.ipynb

```diff
@@ -472,6 +471,17 @@
       "    /* End section1 */\n",
       "    gettimeofday(&end_section1, NULL);\n",
       "    timers->section1 += (double)(end_section1.tv_sec-start_section1.tv_sec)+(double)(end_section1.tv_usec-start_section1.tv_usec)/1000000;\n",
+      "    struct timeval start_section2, end_section2;\n",
```

Comment:
So here we have a new section for BC?

Reply:
loop fusion acting more conservatively => less fusion => one extra section